hack/verify-golint.sh

```diff
@@ -57,29 +57,26 @@ export IFS=$'\n'
 all_packages=(
 	$(go list -e ./... | egrep -v "/(third_party|vendor|staging/src/k8s.io/client-go/pkg|generated|clientset_generated)" | sed -e 's|^k8s.io/kubernetes/||' -e "s|^_${KUBE_ROOT}/\?||")
 )
-linted_packages=(
-	$(cat $linted_file)
+failing_packages=(
```

Comment:
`mapfile -t failing_packages < "$failure_file"` would probably be better. But not in this PR.